pkg/system/phase4_configuring.go

```diff
@@ -777,6 +777,22 @@ func (r *Reconciler) prepareAWSBackingStore() error {
 		// TODO: we need to figure out why secret is not created, and react accordingly
 		// e.g. maybe we are running on azure but our CredentialsRequest is for AWS
 		r.Logger.Infof("Secret %q was not created yet by cloud-credentials operator. retry on next reconcile..", r.AWSCloudCreds.Spec.SecretRef.Name)
+
+		// in case we have a cred request but we do not get a secret
+		minutesSinceCreation := time.Since(r.NooBaa.CreationTimestamp.Time).Minutes()
+		minutesToWait := 3 // this number was chosen arbitrarily
+		if minutesSinceCreation > float64(minutesToWait) {
+			r.Logger.Infof("at least %d minutes passed since noobaa was created, would fallback to create PV Pool backingstore",
```

Comment:
I think we can update the log info something like this
```suggestion
			r.Logger.Infof("at least %d minutes passed since noobaa was created and the secret has not been found yet, would fallback to create PV Pool backingstore",
```

Reply:
Sure, thanks.